app/models/null_meeting.rb

```diff
@@ -0,0 +1,5 @@
+class NullMeeting
+    def held_on
+        "unspecified"
```

Comment:
This may cause issues because it's not the same data type that the `held_on` method returns in `Meeting#held_on`